source/_docs/configuration/templating.markdown

```diff
@@ -238,24 +241,31 @@ The same thing can also be expressed as a filter:
 - `area_id(lookup_value)` returns the area ID for a given device ID, entity ID, or area name. Can also be used as a filter.
 - `area_name(lookup_value)` returns the area name for a given device ID, entity ID, or area ID. Can also be used as a filter.
 
+<div class='note'>
+
+Recent versions of Home Assistant generate the area ID from the area name. So Living Room becomes `living_room`.
+Before the area IDs were automatically generated and may look like this: `deadbeefdeadbeefdeadbeefdeadbeef`.
```

Comment:
We don't keep a changelog in our documentation.

Reply:
Agreed, but this is only acknowledging that for some (most?) users the area id will be a generated identifier.
You have a suggestion?

Reply:
Yes, remove it, as we only reflect the latest state in our documentation.